### PR TITLE
build: Use publishRsync plugin

### PR DESCRIPTION
**Pull request by raboof**
somewhat related to #3304

### CODE REVIEW

File: scripts/release-train-issue-template.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 - [ ] Notify Telemetry / Play team to check against staged artifacts
 - [ ] Run a test against the staging repository to make sure the release went well, for example by using https://github.com/akka/akka-http-quickstart-scala.g8 and adding the sonatype staging repo with `resolvers += "Staging Repo" at "https://oss.sonatype.org/content/repositories/comtypesafe-xxx"`
 - [ ] Release the staging repository to Maven Central.
-- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.9 "deployRsync akkarepo@gustav.akka.io"` to deploy API and reference documentation.
+- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.9 publishRsync` to deploy API and reference documentation.
```
Comment (jrudolph) [inline scripts/release-train-issue-template.md:53]: Do we still want to fix that to 2.12.9 Scala version?

File: project/Publish.scala [hunk 2]
```diff
@@ -31,32 +32,7 @@ object Publish extends AutoPlugin {
...
-    deployRsync := {
-      val (_, host) = (Space ~ StringBasic).parsed
-      deployRsyncArtifact.value.foreach {
-        case (from, to) =>
-          val result = Seq("rsync", "-rvz", s"$from/", s"$host:$to").!
-          require(result == 0, "rsync should return success")
-      }
-    }
+    bintrayRepository := (if (isSnapshot.value) "snapshots" else "maven"),
+    publishRsyncHost in ThisBuild := "akkarepo@gustav.akka.io"
```
Comment (jrudolph) [inline project/Publish.scala:36]: Does that really belong here? Maybe move it to the `docs` module in `build.sbt`.

File: scripts/release-train-issue-template.md (hunk 1 above)
Comment (raboof) [reply scripts/release-train-issue-template.md:53]: I don't see why (but perhaps let's change that in a separate PR)

**Comment by jrudolph** [comment]
> > LGTM, have you already tried it with the RC1?
> 
> I ran it from this branch and it seemed to correctly update the 'snapshot' version of the docs and api/japi references

Great, thanks :)

**Pull request merged**